### PR TITLE
Update debugging_protected_audience_api_services.md

### PR DESCRIPTION
**Pull request by mihnjong-l**
- Updated the data flow diagram.
- Added how to enable the consented debugging.
- Added two screenshots.

### CODE REVIEW

**Comment by chatterjee-priyanka** [comment]
LGTM.

@akshaypundle Would you be able to LGTM here as well? Thanks.

**Review by akshaypundle — APPROVED** [review]
LGTM

File: debugging_protected_audience_api_services.md [hunk 1]
```diff
@@ -55,57 +59,86 @@ misbehaving code and performance problems. This also means there is no need to
...
+    in the future.
+
+### Configure TEE Servers
+
+To enable the AdTech consented debugging, every Protected Audience API TEE
+server should run with the following runtime flags:
+
+```
+ENABLE_OTEL_BASED_LOGGING = "true"
+CONSENTED_DEBUG_TOKEN = "[[secret debug token]]"
```
Comment (akshaypundle) [inline debugging_protected_audience_api_services.md:127]: May help to clarify what the config should be here. Eg. if debug token is "hello" in chrome, then does this need to be configured with "[[hello]]"?

Comment (mihnjong-l) [reply debugging_protected_audience_api_services.md]: Changed to "secret debug token".

**Review by akshaypundle — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**